static/js/public/accordion.js

```diff
@@ -9,9 +9,10 @@
   @param {Boolean} show Whether to show or hide the accordion panel.
 */
 export const toggleAccordion = (element, show) => {
+  console.log("toggleAccordion", element, show);
   element.setAttribute("aria-expanded", show);
   document
-    .querySelector(element.getAttribute("aria-controls"))
+    .querySelector(`#${element.getAttribute("aria-controls")}`)
```

Comment:
could just use `document.getElementById(element.getAttribute("aria-controls"))` ?

Reply:
I'd say that using `querySelector` is preferable. I think we should standardise on that as a code style thing.

Reply:
FINE